x-pack/plugins/infra/public/containers/logs/log_entries/index.ts

```diff
@@ -78,7 +78,7 @@ export interface LogEntriesStateParams {
 }
 
 export interface LogEntriesCallbacks {
-  fetchNewerEntries: () => Promise<TimeKey | null | undefined>;
+  fetchNewerEntries: DebouncedFunc<() => Promise<TimeKey | null | undefined>>;
```

Comment:
The fact that this function might have debouncing behavior should be an implementation detail. Why do we need to expose this in the API here?